integration_tests/test_new_eo_staclike_metadata_fields.py

```diff
@@ -0,0 +1,441 @@
+import yaml
+from datetime import datetime
+from datacube.model.fields import get_dataset_fields, parse_dataset_field
+from datacube.model import Range
+import pytest
+import toolz
+import uuid
+
+from pathlib import Path
+from yaml import CSafeLoader as Loader, CSafeDumper as Dumper
+
+EO_STACLIKE_METADATA_DOC = yaml.load('''---
+name: eo_stac_like_product
+description: Stack_datasets
+dataset:
+  id: [id]
+  creation_dt: ['properties', 'odc:processing_datetime']
+  label: [title]
+  measurements: [measurements]
+  grid_spatial: []
```

Comment:
we need `grid_spatial` to point to some place real

Reply:
Updated tests to point `grid_spatial` to `geometric` reference points.